.github/workflows/loom.yml

```
@@ -95,3 +95,19 @@ jobs:
         working-directory: tokio
         env:
           SCOPE: ${{ matrix.scope }}
+
+  loom-util-sync:
+    name: loom tokio-util::sync
+    # base_ref is null when it's not a pull request
+    if: github.repository_owner == 'tokio-rs' && (contains(github.event.pull_request.labels.*.name, 'R-loom-util-sync') || (github.base_ref == null))
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Install Rust ${{ env.rust_stable }}
+        uses: dtolnay/rust-toolchain@master
+        with:
+            toolchain: ${{ env.rust_stable }}
+      - uses: Swatinem/rust-cache@v2
+      - name: run tests
+        run: cargo test --lib --release --features full -- --nocapture sync::tests
```

```suggestion
        run: cargo test --lib --release --features full -- --nocapture
```

https://github.com/tokio-rs/tokio/blob/9255d96b1b5d6ac85e348d718ed69ea2f80b585c/tokio-util/src/either.rs#L215-L236

It should be ok to exclude these tests from loom, and then run all unit tests using loom.